<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSettings</class>
 <widget class="QDockWidget" name="CameraSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>430</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Camera settings</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QGroupBox" name="groupConstCamSettings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>460</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Constant camera settings</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Select camera (if available)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboCameraName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>55</y>
       <width>420</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelCamPixSize">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera pixel size</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textCamPixSizeX">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="textCamPixSizeY">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>100</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelCamPixSizex">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>35</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelCamPixSizemm">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>100</y>
       <width>35</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="textMaxCamResX">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxCamResx">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>35</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxCamRes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Max. camera resolution</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxCamResmm">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>140</y>
       <width>35</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>px</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="textMaxCamResY">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>140</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelFocalLength">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Focal length</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textFocalLength">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelFocalLengthmm">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>220</y>
       <width>35</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="textCamMatrixSizeX">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="textCamMatrixSizeY">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>180</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelCamMatrixSizemm">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>180</y>
       <width>35</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelCamMatrixSize">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera matrix size</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCamMatrixSizex">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>35</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupDynamixCamSettings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Dynamic camera settings</string>
    </property>
    <widget class="QLineEdit" name="textCurrentCamResX">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="textCurrentCamResY">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelCurrentCamResmm">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>30</y>
       <width>35</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>px</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelCurrentCamRes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Curr. camera resolution</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCurrentCamResx">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>35</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="butApply">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
